x-pack/plugins/ingest_pipelines/public/application/components/pipeline_form/pipeline_form.tsx

```diff
@@ -157,10 +157,11 @@ export const PipelineForm: React.FunctionComponent<PipelineFormProps> = ({
   */
   useUnsavedChangesPrompt({
     titleText: i18n.translate('xpack.ingestPipelines.form.unsavedPrompt.title', {
-      defaultMessage: `Exit pipeline creation without saving changes?`,
```

Comment:
The prompt is not specific to creation (also applies on edit) so I removed this text.